docs/pbc/all/accessibility-improvement/202410.0/b2b/install-the-spryker-accessibility-improvements.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Install the Spryker core keyboard accessibility improvements
+description: Learn how to the Spryker core keyboard accessibility improvements.
+last_updated: March 31, 2025
+template: feature-integration-guide-template
+---
+
+Keyboard accessibility improvements include the following changes:
+
+- Adjusted multiple components to be more accessible via keyboard:
+  - Added `aria-label` attributes to the multiple components to improve screen reader support
+  - Adjusted `tabIndex` attributes to improve keyboard navigation
+- Add a new attribute `parent-class-name` for the `suggest-search` component that helps to improve keyboard accessibility for the product searching process.
+- Improved color schema for better text to background contrast
+
+To install Yves keyboard accessibility improvements, take the following steps:
+
+1. Update modules:
+  - Update the `spryker-shop/catalog-page` module to version `1.29.0` or later.
+  - Update the `spryker-shop/company-page` module to version `1.32.0` or later.
+  - Update the `spryker-shop/customer-page` module to version `1.58.0` or later.1.2
+  - Update the `spryker-shop/product-group-widget` module to version `1.11.1` or later.
+  - Update the `spryker-shop/product-review-widget` module to version `1.16.2` or later.
+  - Update the `spryker-shop/shop-ui` module to version `1.84.0` or later.
+* To update necessary modules run the following command:
+```bash
+  composer update "MODULE_NAME"
+```
+
+2. Adjust templates and files on the project lvl.
+   - Add `skip-link` and ID attribute (href for `skip-link`) to 
+     - src/Pyz/Yves/CatalogPage/Theme/default/templates/page-layout-catalog/page-layout-catalog.twig
+     - src/Pyz/Yves/CompanyPage/Theme/default/templates/page-layout-company/page-layout-company.twig
+     - src/Pyz/Yves/CustomerPage/Theme/default/templates/page-layout-customer/page-layout-customer.twig
+    - Adjust `src/Pyz/Yves/ProductImageWidget/Theme/default/components/molecules/image-gallery/image-gallery.ts`
```

Comment:
It doesn't work. We need to describe it, or maybe we can link a reference PR, but we haven't done it before. We can discuss it with TW/AA probably.